app/jobs/session_trim_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SchoolDataImporterJob < CronJob
+  self.cron_expression = "0 0 1 * *"
+
+  queue_as :session_trim
+
+  def perform
+    Rails.logger.info "Trimming session store..."
+
+    cutoff_period = ENV.fetch("SESSION_DAYS_TRIM_THRESHOLD", 30).to_i.days.ago
+
+    ActiveRecord::SessionStore::Session
+      .where("updated_at < ?", cutoff_period)
+      .delete_all
+  end
+end
```

Comment:
This job is to run daily to remove stale session objects. We might need to consider how this works with our session timeout - do we even have any?

Reply:
I am pretty sure we have an expiration on all cookies, including session cookies. I don't remember exactly how long they last, a week?

Reply:
Two weeks